python/tank/pipelineconfig_factory.py

```diff
@@ -217,12 +213,12 @@ def _from_path(path, force_reread_shotgun_cache):
         if config_context_path not in local_pc_paths:
             raise TankError("You are trying to start Toolkit using the configuration and tank command "
                             "located in '%s'. The path '%s' you are trying to load is not "
-                            "associated with that configuration. The path you are trying to load "
-                            "is associated with the following configurations: %s. "
+                            "associated with that configuration. Instead, it is "
+                            "associated with the following configurations: %s. "
                             "Please use the tank command or Toolkit API in any of those "
                             "locations in order to continue. This error can occur if you "
-                            "have moved a Configuration on disk without correctly updating "
-                            "it. It can also occur if you are trying to use a tank command "
+                            "have moved a configuration on disk without correctly updating "
+                            "it in Shotgun. It can also occur if you are trying to use a tank command "
                             "associated with Project A to try to operate on a Shot or Asset that "
                             "that belongs to a project B." % (config_context_path, path, local_pc_paths))
 
```

Comment:
Personally, I am not totally sure about using smilies in error messages. This may piss people off if they are sitting tearing their hair when nothing is working and they don't understand how to fix it. I have tweaked the message to include a blurb about `tank move_configuration`. Curious to hear what you think.